src/main/java/rcms/utilities/daqaggregator/data/FEDBuilderSummary.java

```diff
@@ -192,7 +187,8 @@ public void calculateDerivedValues() {
 	public int hashCode() {
 		final int prime = 31;
 		int result = 1;
-		result = prime * result + deltaEvents;
+		result = prime * result + ((daq == null) ? 0 : daq.hashCode());
```

Comment:
This will cause cyclical dependency when you call daq.hashCode():
daq.hashCode() -> fedBuilderSummary.hashCode() -> daq.hashCode() -> etc...

I think we once discussed this problem. Please pay attention on this when generating hashCode and equals methods.